chalice/awsclient.py

```diff
@@ -79,7 +80,7 @@ def create_function(self,
         # type: (...) -> str
         kwargs = {
             'FunctionName': function_name,
-            'Runtime': 'python2.7',
+            'Runtime': DEFAULT_LAMBDA_RUNTIME,
```

Comment:
I think it should always be included in both the CreateFunction (which it is required) and the UpdateFunction (which it is should be required based on the PUT semantics we want). Whether it is through a default or making it required, I do not have too strong of an opinion. What are the downsides with making it default? Is that you may be unknowingly change the runtime if you omit it?